pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -136,29 +136,30 @@ func (c *Cluster) Start() error {
 				return fmt.Errorf("failed to update mgr deployment %s. %+v", resourceName, err)
 			}
 		}
-	}
 
-	if err := c.configureOrchestratorModules(); err != nil {
-		logger.Errorf("failed to enable orchestrator modules. %+v", err)
-	}
+		if err := c.configureOrchestratorModules(); err != nil {
+			logger.Errorf("failed to enable orchestrator modules. %+v", err)
+		}
 
-	if err := c.enablePrometheusModule(c.Namespace); err != nil {
-		logger.Errorf("failed to enable mgr prometheus module. %+v", err)
-	}
+		if err := c.enablePrometheusModule(c.Namespace); err != nil {
+			logger.Errorf("failed to enable mgr prometheus module. %+v", err)
+		}
 
-	if err := c.configureDashboard(c.dashboardPort()); err != nil {
-		logger.Errorf("failed to enable mgr dashboard. %+v", err)
-	}
+		if err := c.configureDashboard(mgrConfig); err != nil {
```

Comment:
Do we need to configure the dashboard for each mgr individually? I thought the settings would apply to all the mgr instances if they were specified as `mgr.` instead of `mgr.a`.

Reply:
The problem here is the listening ip adress:
```
**  mgr          advanced mgr/orchestrator_cli/orchestrator  rook           *  
    mgr.a      advanced mgr/dashboard/server_addr          100.192.10.140 *  
    mgr.a      advanced mgr/dashboard/server_port          7000           *  
    mgr.a      advanced mgr/dashboard/ssl                  false          *  
    mgr.a      advanced mgr/prometheus/server_addr         100.192.10.140 *  
**
rook-ceph-mgr-a-97f6d7c8d-kqhqp                                1/1     Running             0          6d16h   100.192.10.140
```

So if you have a mgr.b it should use it's own ip, right?

Reply:
Setting the `server_addr` is actually done now by an init container to the mgr pod (thanks to @BlaineEXE), so I believe the other settings would all be common across all the mgrs

Reply:
They don't seem to actually work as a common config for all mgrs, see https://github.com/rook/rook/issues/2875#issuecomment-484871516 .

Reply:
@q3k Good to know that fixed it for you. Then it makes sense to go ahead and set them for each mgr.